SECURITY.md

```diff
@@ -1,6 +1,61 @@
 # Security Policy
 
-## Reporting a Vulnerability
+## Reporting OSS Vulnerabilities
 
-Please report security vulnerabilities by email to `security@langchain.dev`.
-This email is an alias to a subset of our maintainers, and will ensure the issue is promptly triaged and acted upon as needed.
+LangChain is partnered with [huntr.dev](https://huntr.com/) to provide 
```

Comment:
Instead of huntr.dev you can say huntr / huntr.com or huntr by Protect AI up to you :)